core/src/main/java/io/opencensus/stats/ViewManager.java

```diff
@@ -28,7 +32,49 @@
   public abstract void registerView(View view);
 
   /**
-   * Returns the current stats data, {@link ViewData}, associated with the given view name.
+   * Push model for stats. Registers a {@link View} that will collect data to be accessed
+   * via {@link Handler#export(Collection)} of the passed-in {@link Handler}s. Though it's
+   * also possible to manually get stats data via {@link #getView(View.Name)}.
+   */
+  public abstract void registerView(View view, List<? extends Handler> handlers);
+
+  /**
+   * Set the interval for exporting {@code ViewData} to the given {@code Duration}.
+   *
+   * <p>By default, {@code ViewData} will be pushed to {@code Handler}s every 10 seconds.
+   *
+   * @param duration the interval for exporting stats.
+   */
+  public abstract void setExportInterval(Duration duration);
+
+  /**
+   * Returns the current stats data, {@link ViewData}, associated with the given
+   * {@link View}.
    */
   public abstract ViewData getView(View.Name view);
+
+  /**
+   * An abstract class that registers {@code View} and exports recorded {@code ViewData} in their
+   * own format.
+   */
+  public abstract static class Handler {
+
+    /**
+     * Register a {@code View}. Each {@code View} must be registered first, so that its stats can be
+     * exported appropriately.
+     *
+     * @param view the {@code View} to be registered.
+     */
+    public abstract void registerView(View view);
```

Comment:
Could this method go in the StackDriver exporter?  It will already need to know which views to export in order to call ViewManager.registerView.